resources/puppetlabs/lein-ezbake/staging-templates/project_data.yaml.mustache

```diff
@@ -25,6 +25,8 @@ templates:
     target: ext/debian/{{{project}}}.default
   - source: ext/debian/ezbake.init.erb
     target: ext/debian/{{{project}}}.init
+  - source: ext/debian/ezbake.service.erb
+    target: ext/debian/{{{project}}}.service
```

Comment:
Gah! I noted that we were calling install.sh in the rules file, so I assumed it was doing the right thing :( this sucks